lib/basedriver/commands/execute.js

```diff
@@ -0,0 +1,102 @@
+import _ from 'lodash';
+import log from '../logger';
+import vm from 'vm';
+import { attach } from 'webdriverio';
+import { MJSONWP_ELEMENT_KEY, W3C_ELEMENT_KEY } from '../../protocol/protocol';
+
+const BASE_PATH = '/wd/hub'; // TODO defining this here is brittle but it is hardcoded in routes, so...
+
+let commands = {};
+
+/*
+ * TODO fill out entire docstring
+ *
+ * This method takes a string which is executed as javascript in the context of
+ * a new nodejs VM, and which has available a webdriverio driver object, having
+ * already been attached to the currently running session.
+ */
+commands.executeDriverScript = async function (script, scriptType) {
```

Comment:
should scriptType have a default value here?

Reply:
i need to read the router code but i think it will pass in a value here regardless of what we do?